### PR TITLE
Run flake8 to find syntax errors & undefined names

### PR DESCRIPTION
**Pull request by cclauss**
https://travis-ci.org/ContextLab/hypertools/jobs/370859623#L1014-L1051

### CODE REVIEW

**Comment by jeremymanning** [comment]
Hi @cclauss-- thanks for the pull request!  It looks like the tests are failing; see [here](https://travis-ci.org/ContextLab/hypertools/builds/370782430?utm_source=github_status&utm_medium=notification) for specifics.  Let me or @andrewheusser know if we can help!

**Comment by cclauss** [comment]
This PR is adding tests an those tests are causing the build to fail.  I do not know how to fix the bugs.

**Comment by jeremymanning** [comment]
@cclauss ok, we'll look into this

File: hypertools/tools/procrustes.py [hunk 1]
```diff
@@ -66,15 +65,8 @@ def fit(source, target):
 
         for i, ds in enumerate((source, target)):
             data = ds
-            if _demean:
-                if i == 0:
-                    mean = _offset_in
-                else:
-                    mean = data.mean(axis=0)
-                data = data - mean
```
Comment (cclauss) [inline hypertools/tools/procrustes.py]:
```python
mean = _offset_in if i == 0 else data.mean(axis=0)
data -= mean
```

Comment (andrewheusser) [reply hypertools/tools/procrustes.py]: _demean is always False so i removed the whole statement

Comment (andrewheusser) [reply hypertools/tools/procrustes.py]: oop, i did it wrong though, hang on

**Comment by cclauss** [comment]
Nice!

**Comment by andrewheusser** [comment]
looks good to me!

**Pull request merged**